tests/test_prove.rs

```diff
@@ -70,6 +72,19 @@ fn assert_proof_eq(proof: Proof, expected_proof: Proof) {
     assert_eq!(proof.proof_hex, expected_proof.proof_hex);
 }
 
+#[test]
+fn assert_conflict_on_usage_of_with_and_no_bootloader() {
+    let mut command = std::process::Command::new("stone-prover-cli");
```

Comment:
This test fails because `stone-prover-cli` is not in the PATH. There is a rstest fixture that deals with this, see `#[from(cli_in_path)] _path: (),` in the test just below. You need your test to be a `#[rstest]` instead of `#[test]` though.